### PR TITLE
#1059 added test for People

### PR DESCRIPTION
**Pull request by krzyk**
#1059 
- added links test for People (all other methods are already tested)
- removed People exclusion from jacoco
- removed todo

### CODE REVIEW

**Comment by 0crat** [comment]
Job [#1221](https://github.com/zerocracy/farm/issues/1221) is now in scope, role is `REV`

**Comment by 0crat** [comment]
This pull request [#1221](https://github.com/zerocracy/farm/issues/1221) is assigned to @carlosmiranda[/z](https://www.0crat.com/u/carlosmiranda), here is [why](https://www.0crat.com/footprint/C3NDPUA8L/1731473309); the budget is 15 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§27](http://www.zerocracy.com/policy.html#27) and when you decide to accept the changes, inform @g4s8[/z](https://www.0crat.com/u/g4s8) (the architect) right in this ticket; if you decide that this PR should not be accepted ever, also inform the architect; this [blog post](http://www.yegor256.com/2015/02/09/serious-code-reviewer.html) will help you understand what is expected from a code reviewer; there will be a monetary reward for this job

File: src/test/java/com/zerocracy/pmo/PeopleTest.java [hunk 1]
```diff
@@ -459,6 +456,31 @@ public void throwIfTryToSetEmptyDetails() throws Exception {
...
+        final String srel = "slack";
+        final String jrel = "jira";
+        final String salias = "U67WE3343P";
+        final String jalias = "https://www.0crat.com/jira";
+        people.link(uid, srel, salias);
+        people.link(uid, jrel, jalias);
+        final String format = "%s:%s";
+        MatcherAssert.assertThat(
+            people.links(uid),
+            Matchers.hasItems(
```
Comment (carlosmiranda) [inline src/test/java/com/zerocracy/pmo/PeopleTest.java]: @krzyk let's use [Matchers.containsInAnyOrder](http://hamcrest.org/JavaHamcrest/javadoc/1.3/org/hamcrest/Matchers.html#containsInAnyOrder(T...)) which also ensures iterable length as well as presence of given items.

**Comment by krzyk** [comment]
@carlosmiranda updated, please check

**Comment by g4s8** [comment]
@rultor merge

**Comment by 0crat** [comment]
@ypshenychka[/z](https://www.0crat.com/u/ypshenychka) please review this job completed by @carlosmiranda[/z](https://www.0crat.com/u/carlosmiranda), as in [§30](http://www.zerocracy.com/policy.html#30); the job will be fully closed and all payments will be made when the quality review is completed

**Comment by 0crat** [comment]
The job [#1221](https://github.com/zerocracy/farm/issues/1221) is now out of scope

**Comment by 0crat** [comment]
Payment to `ARC` for a closed pull request, as in [§28](http://www.zerocracy.com/policy.html#28): +10 point(s) just awarded to @g4s8[/z](https://www.0crat.com/u/g4s8)

**Comment by ypshenychka** [comment]
@carlosmiranda According to our [QA Rules](http://www.zerocracy.com/policy.html#42):
>The code reviewer found at least three problems in the code.
Comments were mostly about design problems, not cosmetic issues.

 Only one issue found during code review. 
Please confirm that you'll try to find at least three major problems while future reviews.

**Comment by carlosmiranda** [comment]
@ypshenychka confirmed

**Comment by ypshenychka** [comment]
@carlosmiranda Thanks

**Comment by ypshenychka** [comment]
@0crat quality acceptable

**Comment by 0crat** [comment]
Order was finished, quality is "acceptable": +15 point(s) just awarded to @carlosmiranda[/z](https://www.0crat.com/u/carlosmiranda)

**Comment by 0crat** [comment]
Quality review completed: +8 point(s) just awarded to @ypshenychka[/z](https://www.0crat.com/u/ypshenychka)

**Pull request merged**